src/pages/download.jsx

```diff
@@ -10,14 +10,20 @@ export default function Home() {
             title={siteConfig.title}
             description="Download page for the conda-forge installer"
         >
-            <main>
-                <h1 style={{
-                    marginTop: 10, textAlign: "center"
-                }}>
-                    Download the conda-forge Installer
-                </h1>
+            <div className="container margin-vert--lg" >
+                <main>
+                    <h1>
+                        Download the conda-forge Installer
+                    </h1>
+                    <p>
+                        Miniforge is a minimal installer for a conda-forge-based distribution, including conda, mamba, and their dependencies.
+                    </p>
+                    <p>
+                        If more packages are needed, use the <code>conda install</code> or <code>mamba install</code> command to install from the thousands of packages available in the conda-forge distribution.
```

Comment:
I wonder if this may seems to suggest that it is a good idea to install packages in the base environment. Perhaps we should mention `conda create` and `mamba create` so it is more clear that it is recommended to create specific environments instead of installing them in the `base` env?

Reply:
We could add a separate sentense about creating isolated environments.

Reply:
Then resolving this suggestion. Will add another phrase about this.

Reply:
Sure!